### DISCUSSION TITLE
What is the replacement for mg_str_starts_with()?

### DISCUSSION THREAD

**Discussion by sebhub** [cesanta/mongoose#1516] [Ask questions, get answers] [answered]
In Mongoose 6, there is a `mg_str_starts_with()` function. What is the replacement for this function in Mongoose 7?

**Comment by cpq** [comment, marked as answer]
Please use https://mongoose.ws/documentation/#mg_match

**Reply by sebhub** [reply]
The `mg_match()` involves patterns. The `mg_str_starts_with()` is a lot simpler which less runtime overhead.
